suites/upgrade/infernalis/older/0-cluster/start.yaml

```diff
@@ -0,0 +1,24 @@
+meta:
+   - desc: |
+      white-list items
+      assign cluster nodes
```

Comment:
@jdurgin incorporated ^ (I personally feel that descriptions already useful as I learn more about the tests :) )